src/plugins/intel_cpu/tests/functional/subgraph_tests/src/inplace_edge.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
+#include "ov_models/utils/ov_helpers.hpp"
+#include "ov_models/builders.hpp"
+
+using namespace CPUTestUtils;
+using namespace InferenceEngine;
+
+namespace SubgraphTestsDefinitions {
+// Subgraph:
+/*
+ *            Constant  Constant
+ *                 \    /
+ *                  \  /
+ *               Transpose
+ *  Parameter      /   \     Parameter
+ *        \       /     \       /
+ *         \     /       \     /
+ *         Eltwise       Eltwise
+ *               \       /  
+ *                Eltwise
+ *                   |
+ *                 Result
+ */
+
+class NonInputInPlaceTest : public testing::WithParamInterface<InferenceEngine::Precision>,
+                            virtual public LayerTestsUtils::LayerTestsCommon {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<InferenceEngine::Precision> obj) {
+        std::ostringstream result;
+        result << "NonInputInPlaceTest" << obj.param.name();
+        return result.str();
+    }
```

Comment:
Please add the precision to the name, otherwise it's impossible to distinguish the tests from the test set.

Reply:
fixed